### PR TITLE
reduce code duplication

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2/util/Util.cpp [hunk 1]
```diff
@@ -590,6 +590,44 @@ money32 add_clamp_money32(money32 value, money32 value_to_add)
...
+
+float flerp(float a, float b, float t)
+{
+    if (t <= 0) return a;
+    if (t >= 1) return b;
+
+    float range = b - a;
+    float amount = range * t;
+    return a + amount;
+}
```
Comment (IntelOrca) [inline src/openrct2/util/Util.cpp]: `flerp` is only used in `lightfx.c`.

Comment (janisozaur) [reply src/openrct2/util/Util.cpp]: I moved it here to not split lerp and flerp between two separate units.

Do you want me to move flerp back?

Comment (IntelOrca) [reply src/openrct2/util/Util.cpp]: I'm not bothered, just mentioned it in case you didn't realise.

**Comment by janisozaur** [comment]
Merged as b574c2317, but pushed directly to `develop` to fix the typo I spotted in commit message.

**Pull request closed without merging**